jobs/JGLOBAL_WAVE_POST_BNDPNT

```diff
@@ -9,9 +9,14 @@ export errchk=${errchk:-err_chk}
 export MP_PULSE=0
 
 # Set COM Paths and GETGES environment
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_WAVE_PREP COM_WAVE_HISTORY COM_WAVE_STATION
-
-if [[ ! -d ${COM_WAVE_STATION} ]]; then mkdir -p "${COM_WAVE_STATION}"; fi
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+   COMIN_WAVE_PREP:COM_WAVE_PREP_TMPL \
+   COMOUT_WAVE_HISTORY:COM_WAVE_HISTORY_TMPL \
```

Comment:
what is being written out in `COMOUT_WAVE_HISTORY` from this job?
This should be a `COMIN` variable, if I understand this job correctly.

Reply:
The logic was derived from [here](https://github.com/HenryWinterbottom-NOAA/global-workflow/blob/526c64ccd997e316a6eba781fd14cacaf41ba897/jobs/JGLOBAL_WAVE_POST_BNDPNT#L14).

Reply:
@aerorahul What is the verdict here?

Reply:
What are the inputs derived from in this job?
Those should be labeled `COMIN_`.
What are the outputs from this job?
Those should be labeled as `COMOUT_`.
Without looking in what the job is doing and placing data, I don't think I can state definitively.

Reply:
Updated.

Reply:
@HenryWinterbottom-NOAA I still do not think we need a COMOUT_WAVE_HISTORY.  If that means that we remove it from line 18 as well, I think that's okay.

Reply:
Jessica is correct. I did an audit of these last week, `COMOUT_WAVE_HISTORY` should not be needed.

Reply:
@JessicaMeixner-NOAA Thank you. I missed this in the thread. Fixed.